src/main/java/com/spotify/avro/mojo/SchemaParserBuilder.java

```diff
@@ -0,0 +1,7 @@
+package com.spotify.avro.mojo;
+
+import org.apache.avro.Schema;
+
+public interface SchemaParserBuilder {
+  Schema.Parser build();
```

Comment:
Why not keeping the `withTypes`, `withValidate` and `withValidateDefaults` since all `Schema.Parser` have to support those ?

Reply:
When users implement their own builder, they most likely set those values themselves within it.
The default implementation still has ability for setting validations.
This approach allows for smaller api footprint allowing more flexibility how users want to implement builder, and not restricting us to future changes in Avro API (adding more validations, removing some?).

Reply:
Then again - adding those function would allow easier control over validations for users if org infra team would provide custom builder (as I imagine it would be for us at Spotify :))